Mathlib/RingTheory/LaurentSeries.lean

```diff
@@ -771,7 +771,7 @@ theorem Cauchy.eventually_mem_nhds {ℱ : Filter (LaurentSeries K)} (hℱ : Cauc
     rw [← WithZero.coe_unzero γ.ne_zero, WithZero.coe_lt_coe, hD₀, neg_neg, ofAdd_sub,
       ofAdd_toAdd, div_lt_comm, div_self', ← ofAdd_zero, Multiplicative.ofAdd_lt]
     exact zero_lt_one
-  apply coeff_eventually_equal hℱ |>.mono
+  apply coeff_eventually_equal (D := D) hℱ |>.mono
```

Comment:
What's the purpose in this change?

Reply:
Without it, Lean was creating a goal for `D` that spanned, that was closed only when the "current" goal was closed, but lingered as a "side goal" for the ride.